docs/DeveloperGuide.md

```diff
@@ -247,36 +247,36 @@ The following activity diagram summarizes what happens when a user enters an `ad
-* Else, the details of the guest created will not have a `Request`.
+* Else, the details of the guest created will not have an intended value for `Request` specified by the user.
 * Finally, the guest is added to the model.
 
 <span style="font-size: large; color: #e46c0a">Design Considerations:</span>
 
 **Aspect: How to deal with duplicate entries**
 
-* **Alternative 1 (current choice):** Reject any entry with a duplicate name or room that is case-insensitive.
-    * Pros: Easy to implement.
+* **Alternative 1 (current choice):** Reject any entry with a duplicate name or room that is case-sensitive.
+    * Pros: Easier to implement.
     * Cons: Will have issues with adding people of the same name or room to GuestBook.
 
 * **Alternative 2:** Allow duplicate entries to be added with no consequences.
     * Pros: Allows for multiple guests of the same name or room to be added.
     * Cons: Easy to abuse the command intentionally (to break the program) or unintentionally (accidentally). By allowing duplicate rooms, 
-the user might allocate the two different guests to the same room. 
+the user might allocate the two different guests to the same room.
 
 In small hotels, the chances of getting two guests that have identical full names are
 incredibly unlikely. Also, having two different guests allocated to the same room is a rather disastrous mistake and should be avoided. With these considerations in mind, we decided to proceed with Alternative 1.
 
 **Aspect: Excluding `isRoomClean` field in the `add` command**
 * As adding a guest will be done during check in, the room should by default be cleaned for the guest to immediately
-be able to enter their room. Hence, we chose to remove `isRoomClean` as it would make the `add` command unnecessarily longer.
+be able to enter their room. Hence, we chose to exclude `isRoomClean` as it would make the `add` command unnecessarily longer. This enables the user to enter their inputs with greater efficiency.
 
 **Aspect: Excluding `Bill` field in the `add` command**
 * As adding a guest will be done during check in, the guest should not have incurred any bill costs yet. Hence, we set
-the initial bill to be 0 and chose to remove `Bill` as it would make the `add` command unnecessarily longer.
+the initial bill to be 0 and chose to exclude `Bill` as it would make the `add` command unnecessarily longer. This enables the user to enter their inputs with greater efficiency.
 
 **Aspect: Making `Request` optional in the `add` command**
 * As adding a guest will be done during check in, the guest may or may not have any special requests to make for the room. Hence,
-we chose to make `Request` optional and default it as blank should it not be provided. This would also reduce the length
+we chose to make `Request` optional and **default its value as a hyphen (`-`)** should it not be provided. This would also reduce the length
```

Comment:
Should this be "default its value to a hyphen"?

Reply:
Agreed!